src/Widgets/NavigationWidget/SubComponents/CurrentUserDropdown.tsx

```diff
@@ -75,7 +75,8 @@ export const CurrentUserDropdown = connect(function CurrentUserDropdown({
         }}
         to={Obj.root()}
       >
-        <i className={`bi bi-box-arrow-right`}></i> Log out
+        <i className={`bi bi-box-arrow-right`}></i>
+        <ContentTag tag="span" content={widget} attribute="logOutLabel" />
```

Comment:
I would prefer if the `logOutLabel` is a config in the widget properties, not inplace editable.